spec/outputs/example_spec.rb

```diff
@@ -1 +1,20 @@
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/outputs/example"
+require "logstash/codecs/plain"
+require "logstash/event"
+
+describe LogStash::Outputs::Example do
+  let(:sample_event) { LogStash::Event.new }
+
+  context "#receive" do
+    subject do
+      output = LogStash::Outputs::Example.new
+      output.register
```

Comment:
I don't think this will do a real improvement in error traceability in the suite. 
BUT since it give more real world usage of rspec I agree to include this change.